vars/withAcrClient.groovy

```diff
@@ -3,9 +3,7 @@ def call(String subscription, Closure block) {
   if (env.IS_DOCKER_BUILD_AGENT && env.IS_DOCKER_BUILD_AGENT.toBoolean()) {
     doAcrOrKubectlTask(subscription, false, block)
   } else {
-    withDocker('hmcts/cnp-aks-client:az-2.44.1-kubectl-1.26.0-helm-3.10.3', null) {
       doAcrOrKubectlTask(subscription, true, block)
```

Comment:
also no need to set this to true anymore as it's no longer in a container.

you should be able to remove `doAcrOrKubectlTask` as it's no longer needed, and might be able to remove the `alwaysLogin` functionality if it's no longer needed

Reply:
Something like this? https://github.com/hmcts/cnp-jenkins-library/pull/1108/files#diff-5348ea2ed32827d22070457c3804a837f4cbbdd885a6e2d46a2939cb135294dcR2-R5

Reply:
almost, https://github.com/hmcts/cnp-jenkins-library/blob/d841760cfc3a8a0def31ed7405a663ef011b60ff/vars/withSubscriptionLogin.groovy#L8-L15 is no longer used so should be removed